doc/nrf/app_dev/../nrf54l/kmu_provision: correct desc. on `west flash` (#23733)
nrfconnect/sdk-nrf: doc/nrf/app_dev/device_guides/nrf54l/kmu_provision.rst

Alternative provisioning method
*********************************

To simplify the development process, keys can be generated and then provisioned at the same time as the flashing process.

> You can provision keys during the flashing when the build directory contains the :file:`keyfile.json` file with commands such as ``west flash --recover`` or ``west flash --erase``.

Reviewer: ```suggestion
You can provision keys during flashing when the build directory contains the :file:`keyfile.json` file with commands, such as ``west flash --recover`` or ``west flash --erase``.
```

> When flashing a project that contains NSIB, only the ``west flash --recover`` command is available, as the programming file contains UICR provisioning data as well.

Reviewer: ```suggestion
When flashing a project that contains NSIB, you can only use the ``west flash --recover``, as the programming file contains UICR provisioning data as well.
```

> The :file:`keyfile.json` file might be generated during the build process (for example, when running ``west build``), if the :Kconfig:option:`SB_CONFIG_SECURE_BOOT_GENERATE_DEFAULT_KMU_KEYFILE` and/or the :Kconfig:option:`SB_CONFIG_MCUBOOT_GENERATE_DEFAULT_KMU_KEYFILE` options are enabled.

Reviewer: ```suggestion
You can generate the :file:`keyfile.json` file during the build process (for example, when running ``west build``) if the :Kconfig:option:`SB_CONFIG_SECURE_BOOT_GENERATE_DEFAULT_KMU_KEYFILE` or :Kconfig:option:`SB_CONFIG_MCUBOOT_GENERATE_DEFAULT_KMU_KEYFILE` Kconfig options are enabled.
```

Alternative provisioning method
*********************************

To simplify the development process, keys can be generated and then provisioned at the same time as the flashing process.

> It is possible to provision keys during the flashing when the build directory contains the keyfile.json file with commands
> such as `west flash --recover` or `west flash --erase`. When flashing a project which contains NSIB only the `west flash --recover`
> command is available as the programming file contains UICR provisioning data as well.

Reviewer 2: ```suggestion
You can provision keys during the flashing when the build directory contains the :file:`keyfile.json` file with commands such as ``west flash --recover`` or ``west flash --erase``. 
When flashing a project that contains NSIB, only the ``west flash --recover`` command is available, as the programming file contains UICR provisioning data as well.
```

> The `keyfile.json` file might be generated during the build process (e.g., when running `west build`) if the ``SB_CONFIG_SECURE_BOOT_GENERATE_DEFAULT_KMU_KEYFILE`` and/or the ``SB_CONFIG_MCUBOOT_GENERATE_DEFAULT_KMU_KEYFILE`` options at the sysbuild level are enabled.

Reviewer 2: ```suggestion
The :file:`keyfile.json` file might be generated during the build process (for example, when running ``west build``), if the :Kconfig:option:`SB_CONFIG_SECURE_BOOT_GENERATE_DEFAULT_KMU_KEYFILE` and/or the :Kconfig:option:`SB_CONFIG_MCUBOOT_GENERATE_DEFAULT_KMU_KEYFILE` options are enabled.
```

> These options enable respectively the NSIB and the MCUboot keys inclusion into generated keyfile.json.

Reviewer 2: ```suggestion
These options enable, respectively, the NSIB and the MCUboot keys, included in the generated :file:`keyfile.json` file.
```
Please check the meaning.

This file contains the necessary key provisioning information.

If you set the ``SB_CONFIG_SECURE_BOOT_SIGNING_KEY_FILE`` Kconfig option to a PEM key file, that specific file will be used.
If not, the build will use the default key named :file:`GENERATED_NON_SECURE_SIGN_KEY_PRIVATE.pem`, which is located in the build directory.

> Similarly, MCUboot uses the key file designated by the ``SB_CONFIG_BOOT_SIGNATURE_KEY_FILE`` option.

Reviewer 2: ```suggestion
Similarly, MCUboot uses the key file designated by the :Kconfig:option:`SB_CONFIG_BOOT_SIGNATURE_KEY_FILE` option.
```
